apps/wallet/src/ui/app/components/ledger/ConnectLedgerModal.tsx

```diff
@@ -28,17 +28,7 @@ export function ConnectLedgerModal({
     const onContinueClick = async () => {
         try {
             setConnectingToLedger(true);
-
             await connectToLedger();
-
```

Comment:
This actually isn't needed because the next route will show an error toast if the user doesn't have the Sui application open